modular_zubbers/code/modules/events/ev_roleplay_check.dm

```diff
@@ -3,7 +3,7 @@ GLOBAL_LIST_EMPTY(dorms_areas)
 
 /area/station/commons/dorms/New()
 	. = ..()
-	GLOB.dorms_areas += src
+	GLOB.dorms_areas[src] = TRUE
```

Comment:
```suggestion
	GLOB.dorms_areas |= src
```
This doesn't need to be an associative list, `|=` will prevent duplicate entries

Reply:
I changed it to an assoc list  because of a performance benefit for the erp checking.